bundler/benchmarks/bm_bundle_install_rails.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'support/benchmark_bundler.rb'
+require 'bundler/cli'
+
+
+Dir.mktmpdir do |dir|
+  Dir.chdir(dir) do
+    File.write('Gemfile', <<-GEMFILE)
+      source 'https://rubygems.org'
+      gem 'rails', '~> 5.0.0'
+      gem 'nokogiri', '1.6.7.2'
+    GEMFILE
+
+    system("bundle install --quiet")
```

Comment:
Noted. Would you know what the rough % breakdowns would be like 50% network, xx% ___ and xx% ___? Andre suggested using gemstash to reduce network uncertainty, which I'll try this week.

My thought for this one is it's a measure of what the user will experience. It's a sanity check for nothing was changed that causes a significant hit in overall performance without good reason and not for micro optimizations.